lib/takeaway.rb

```diff
@@ -0,0 +1,34 @@
+class Takeaway
+  def initialize(menu_klass:,
+                 menu_loader_klass:,
+                 order_klass:,
+                 order_total_klass:,
+                 notifier_klass:,
+                 config_loader_klass:,
+                 delivery_estimator_klass:)
+    @menu = menu_klass.new(menu_loader_klass: menu_loader_klass)
+    @order = order_klass.new(order_total_klass: order_total_klass)
+    @notifier = notifier_klass.new(config_loader_klass: config_loader_klass)
+    @estimator = delivery_estimator_klass.new
+  end
```

Comment:
You were absolutely correct in the standup - many instance variables and an overly-long parameter list. I feel like this is a symptom of pushing all the dependencies up to the top of the tree.